src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -704,6 +704,9 @@ public void PlaceOrganelle()
             // Only trigger tutorial if something was really placed
             TutorialState.SendEvent(TutorialEventType.MicrobeEditorOrganellePlaced, EventArgs.Empty, this);
         }
+
+        // Disable Continue button if there are islands
+        gui.SetFinishButtonStatus(!HasIslands());
```

Comment:
I said this before, but just for completeness I'll say it here as well: I think that the finish button should not be disabled, instead there should be a popup saying why you can't leave the editor and suggest how to fix it (undoing if you don't have enough MP is the only way).

Reply:
Is disabling the button ok when I highlight the islands in red?
If so, this conversation is resolved